components/Link.jsx

```diff
@@ -1,22 +1,32 @@
 import React, { PropTypes } from 'react'
+import { Link as ReactRouterLink } from 'react-router'
```

Comment:
Right... So another thing we thought about was maybe move a prop named `wrapAnchor` and then wrap the entire thing in a `span` instead of `a` and pass the reactRouter Link as part of the children. That way I am not adding react-router to this project, and not duplicating the component. WDYT?